content/xeus-sqlite/simple-operations.ipynb

```diff
@@ -15,22 +15,42 @@
   "nbformat_minor": 4,
   "nbformat": 4,
   "cells": [
+    {
+      "cell_type": "markdown",
+      "source": "# JupyterLite `xeus-sqlite` Kernel Demo\n\nThe [`jupyterlite/xeus-sqlite-kernel`](https://github.com/jupyterlite/xeus-sqlite-kernel) wraps the original [`jupyter-xeus/xeus-sqlite`](https://github.com/jupyter-xeus/xeus-sqlite/) kernel for use in JupyterLite.\n\nOriginal kernel docs can be found [here](https://xeus-sqlite.readthedocs.io/en/latest/).\n\nThe kernel provides cell magic for command line database operations, and native execution of SQL code against a connected database.",
+      "metadata": {}
+    },
+    {
+      "cell_type": "markdown",
+      "source": "## Creating a Database\n\nLine magic is used to create an in-memory database:",
+      "metadata": {}
+    },
     {
       "cell_type": "code",
       "source": "%CREATE example_db.db",
       "metadata": {
         "trusted": true
       },
-      "execution_count": null,
+      "execution_count": 1,
       "outputs": []
     },
+    {
+      "cell_type": "markdown",
+      "source": "Currently, there is no ability to:\n\n- save the database to browser storage;\n- export the database;\n- load a database from browser storage;\n- load a database from a URL;\n- load a database from the desktop;\n- connect to a remote sqlite database file.",
+      "metadata": {}
+    },
+    {
+      "cell_type": "markdown",
+      "source": "## Create and Populate Tables\n\nTables are created and populated using SQL:",
+      "metadata": {}
+    },
     {
       "cell_type": "code",
       "source": "CREATE TABLE players (Name STRING, Class STRING, Level INTEGER, Hitpoints INTEGER)",
       "metadata": {
         "trusted": true
       },
-      "execution_count": null,
+      "execution_count": 2,
```

Comment:
Let's keep the notebook without execution counts and output?

Reply:
For someone wanting a quick preview of what's possible, showing run cells means folk can just preview the notebooks with outputs without having to run them?

It also shows what's expected if things *don't* work for whatever reason?

But that's my personal preference...

Reply:
ah ok fair enough. Some other notebooks in this repo have cleared outputs so it was mostly to keep things consistent.